### PR TITLE
Add Typhoon to Injection section

### PR DESCRIPTION
**Pull request by miguelarauj1o**
## Project URL
https://github.com/appsquickly/Typhoon

## Description
Add Typhoon to Injection section

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 7 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -1785,6 +1785,7 @@ Most of these are paid services, some have free tiers.
 * [Reliant](https://github.com/appfoundry/Reliant) - Nonintrusive Objective-C dependency injection.
 * [Kraken](https://github.com/sabirvirtuoso/Kraken) - A Dependency Injection Container for Swift with easy-to-use syntax.
 * [Cleanse](https://github.com/square/Cleanse) - Lightweight Swift Dependency Injection Framework by Square. :large_orange_diamond:
+* [Typhoon](https://github.com/appsquickly/Typhoon) - Poweful dependency injection for iOS & OSX (Objective-C & Swift).
```
Comment (dkhamsing) [inline README.md]: 1. Typo here, should say `Powerful`
2. Remove `for iOS & OSX`, iOS is assumed

Thx

Comment (miguelarauj1o) [inline README.md]: Hey @dkhamsing, I just copied from repo description. Thanks for checking!

**Comment by dkhamsing** [comment]
thx!

**Pull request merged**